src/main/webapp/WEB-INF/views/DesignPage.html

```diff
@@ -0,0 +1,344 @@
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+            </button>
+            <a class="navbar-brand" href="#">Great<i class="fa fa-rocket fa-lg" aria-hidden="true">
+                <text/>
+            </i>Start</a>
+        </div>
+        <div class="collapse navbar-collapse" id="responsive-menu">
+            <ul class="nav navbar-nav navbar-left">
+                <li class="active"><a href="#">Home</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Events</a></li>
+                <li><a href="#">Projects</a></li>
+                <li>
+                    <form role="search" class="navbar-form navbar-left">
+                        <div class="form-group">
+                            <input type="text" placeholder="Search" class="form-control">
+                        </div>
+                    </form>
+                </li>
+            </ul>
+            <ul class="nav navbar-nav navbar-right">
+                <li class="active"><a href="#modal-login" data-toggle="modal">Login</a></li>
+            </ul>
+        </div>
+    </div>
+</div>
+<div class="header-wrap">
+    <div class="container">
+        <div class="row centered">
+            <div class="col-lg-8 col-lg-offset-2">
+                <h1 class="title">GreatStart is awesome!</h1>
+                <h2 class="under-title">Join us</h2>
+            </div>
+        </div>
+    </div>
+</div>
+<div class="container column-wrap">
+    <div class="row centered">
+        <div class="col-lg-4 col-md-4 col-sm-4">
+            <i class="fa fa-lightbulb-o fa-3x"></i>
+            <h4>Ideas</h4>
+            <div><strong>Just Text Example</strong></div>
+            <div>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+                dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip
+                ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu
+                fugiat nulla pariatur.
+            </div>
+        </div>
+        <div class="col-lg-4 col-md-4 col-sm-4">
+            <i class="glyphicon glyphicon glyphicon-grain gi-3x"></i>
+            <h4>Help</h4>
+            <div><strong>Just Text Example</strong></div>
+            <div>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+                dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip
+                ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu
+                fugiat nulla pariatur.
+            </div>
+        </div>
+        <div class="col-lg-4 col-md-4 col-sm-4">
+            <i class="fa fa-users fa-3x"></i>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)